README.md

```diff
@@ -35,18 +37,53 @@ To build a single project for production (for instance, the `@coveo/atomic` pack
 npm run build -w @coveo/atomic
 ```
 
+## Development mode
+
+To start Atomic & Headless simultaneously in development (recommended), run:
+
+```sh
+npm run dev:atomic
+```
+
 To start a single project in development (for instance, the `quantic` package), run:
 
 ```sh
-npm dev -w @coveo/quantic
+npm run dev -w @coveo/quantic
```

Comment:
We should probably recommend `npx nx` over `npm run -w`.

Either way, Nx will run necessary tasks before the task we asked it to do. However, the last time I tried, Nx's caching mechanism was including the CWD when determining whether a task should be re-run or taken from the cache. Unfortunately, using `npm run -w` seems to change the CWD and invalidate `npm run build`'s cache.

Here's what this task would look like with Nx:
```suggestion
npx nx run quantic:dev
```

Here's what a task with a colon in its name would look like with Nx:
```bash
npx nx run quantic:"test:watch"
```